nrich-form-configuration-api/src/main/java/net/croz/nrich/formconfiguration/api/service/FormConfigurationService.java

```diff
@@ -35,4 +35,11 @@ public interface FormConfigurationService {
      */
     List<FormConfiguration> fetchFormConfigurationList(List<String> formIdList);
 
+    /**
+     * Returns a list of all registered {@link FormConfiguration} instances.
+     *
+     * @return a list of form configuration instances
+     */
+    List<FormConfiguration> fetchAllFormConfigurationList();
```

Comment:
Not sure about `All` in method name, `fetchFormConfigurationList` without parameters is clear enough to me.

Reply:
This is because it is called findAll in controller and I wanted to be consitent.